test/test_jit.py

```diff
@@ -1720,6 +1724,18 @@ def func(x, y):
         self.assertEqual(x.grad, grad)
         self.assertEqual(y.grad, grad.sum(dim=0))
 
+    def test_sum(self):
+        @torch.jit.script
+        def func(x):
+            return x.sum(dim=[4])
+
+        @torch.jit.script
+        def func2(x):
+            return x.sum(dim=4)
+
+        self.assertExpected(canonical(func.graph), subname='1')
+        self.assertExpected(canonical(func2.graph), subname='2')
```

Comment:
Wouldn't it be simpler to just assert that `canonical(func.graph) == canonical(func2.graph)`?